spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ProjectControllerHelperService.java

```diff
@@ -920,7 +910,7 @@ public JSONObject listWatchOverview(HttpSession session, JSONObject json) {
     try {
       final ProjectOverview overview = requestManager.getProjectOverviewById(overviewId);
       sb.append("<ul class='bullets' style='margin-left: -30px;'>");
-      for (final User theUser : overview.getWatchers()) {
+      for (final User theUser : overview.getProject().getWatchers()) {
```

Comment:
Overview has its own watchers, so this shouldn't go up to the project (bad merge in ProjectOverview)